queue/queue.js

```diff
@@ -29,6 +34,20 @@ var queueApp = angular.module("queueApp", ["ngMaterial", "ngMessages"])
 			}
 		});
 	}
+
+	$scope.getSites = function() {
+		QueueService.getSites().then(function(data) {
+			if(data == null) {
+				console.log('server error');
```

Comment:
Might want to display an error to users when this happens. Correct me if I'm wrong but this would render the page inoperable.

Reply:
Yep, created https://github.com/UNL-CSE-AMBASSADORS/VITA/issues/130 to address this